components/data-bridge/org.wso2.carbon.databridge.core/src/main/java/org/wso2/carbon/databridge/core/DataBridge.java

```diff
@@ -437,6 +444,11 @@ public boolean deleteStreamDefinition(String streamName,
         return streamDefinitionStore.deleteStreamDefinition(streamName, streamVersion, tenantId);
     }
 
+    @Override
```

Comment:
Moved the method to AbstractStreamDefinitionStore in commit 130d9fb481273997a6cead2e90ecc8120406f54c